content/post/082-our-focus-for-2020.md

```diff
@@ -0,0 +1,86 @@
+
+IPFS Cluster also released [“collaborative clusters”](https://blog.ipfs.io/2020-01-09-collaborative-clusters/), a new feature to enable package manager maintainers and mirrors to add and replicate repositories across a community of IPFS nodes. With collaborative clusters, any maintainer can push new updates to the pinset of data to mirror, which is then sharded and sync’d across all mirroring nodes. We’ve already seen package managers like Pac-Man added to collaborative clusters along with many “data package managers” like Wikipedia and Project Gutenberg, [follow these instructions to add your own!](https://collab.ipfscluster.io/)
+
+## How We Did Against Our Goal
+
+We made huge strides this year on the performance at scale needed by many package manager communities, but it’s clear there’s still lots of work left to be done to achieve widespread adoption and meet the needs of more types of package managers. While focusing on a specific use case did help us identify and drive important fixes, it didn’t give us feedback across the entire IPFS ecosystem as to what pain points were top priority blockers to growth or value-add. We also found that a number of paths to increasing IPFS usage in package managers were actually divergent from the core goal we wanted to achieve: making IPFS itself better. This tension required constant vigilance to ensure features and improvements would be felt by many downstream use cases—not just specialized package manager tooling. 
+
+In the end, we succeeded at staying focused, not building another new package manager, and landing a ton of features and improvements to make IPFS better for everyone; where we didn’t succeed was shipping and integrating those improvements into existing package managers to drive adoption and visibility of IPFS with an awesome community of aligned and talented developers. With the features and improvements landing in 0.5.0, that future adoption work is significantly unblocked, however we also learned through our research that most package manager communities aren’t fast adopters of new tooling around package distribution due to the semi-pro-bono nature of maintainership. To support a slower, more ad-hoc adoption cadence, we plan to use channels like [DevGrants](https://github.com/ipfs/devgrants) and Collaborations to support package manager adopters acting within their own communities. This allows us to continue focusing our core working groups on improving and augmenting the core protocol and reference implementation, while supporting many community applications and refinements of these tools. 
```

Comment:
Just to clarify, this is go-ipfs 0.5.0 right?

Reply:
yep! Should I clarify/link again?

Reply:
```suggestion
In the end, we succeeded at staying focused, not building another new package manager, and landing a ton of features and improvements to make IPFS better for everyone; where we didn’t succeed was shipping and integrating those improvements into existing package managers to drive adoption and visibility of IPFS with an awesome community of aligned and talented developers. With the [features and improvements landing in 0.5.0](https://github.com/ipfs/go-ipfs/issues/6776), that future adoption work is significantly unblocked, however we also learned through our research that most package manager communities aren’t fast adopters of new tooling around package distribution due to the semi-pro-bono nature of maintainership. To support a slower, more ad-hoc adoption cadence, we plan to use channels like [DevGrants](https://github.com/ipfs/devgrants) and Collaborations to support package manager adopters acting within their own communities. This allows us to continue focusing our core working groups on improving and augmenting the core protocol and reference implementation, while supporting many community applications and refinements of these tools. 
```

Reply:
Much better!